generator/Cargo.toml

```diff
@@ -5,7 +5,9 @@ authors = ["Maik Klein <maikklein@googlemail.com>"]
 edition = "2018"
 
 [dependencies]
-vk-parse = { version = "0.5.0", features = ["vkxml-convert"] }
+# https://github.com/krolli/vk-parse/pull/17
+# vk-parse = { git = "https://github.com/krolli/vk-parse", rev = "873860cbcacd4a35f489d0d6b6dd7eb24156136a", features = ["vkxml-convert"] }
+vk-parse = { git = "https://github.com/MarijnS95/vk-parse", branch = "enums-bitwidth", features = ["vkxml-convert"] }
```

Comment:
I'll have to maintain this branch until either the PR to `vk-parse` is merged or we fork `vk-parse` elsewhere, perhaps directly under Maik's account.

Reply:
Branch got merged, currently pointing this to `krolli/vk-parse` directly until a release is made :)